src/assets/scss/_recordset-table.scss

```diff
@@ -157,21 +157,22 @@
     }
   }
 
+  // Changing the padding and adding line-height for the action button to align with the rest of the columns in the table
   .action-btns {
     text-align: center;
     font-size: 1.2rem;
-    // since we're increasing the size of icons, we should decrease the padding
-    padding: 5px 0 0 0;
-
+    padding: 3px 0 0 0;
+    line-height: 1.5;
```

Comment:
added line-height as it was not aligning in windows when i tested in saucelabs. this is the closest alignment i could get for the action buttons